src/frontend/src/routes/(app)/+layout.svelte

```diff
@@ -41,7 +42,11 @@
 
 	<AuthGuard>
 		<SplitPane>
-			<NavigationMenu slot="menu" />
+			<NavigationMenu slot="menu">
+				{#if route === 'tokens'}
+					<DappsCarousel styleClass="w-80 xl:block hidden" />
```

Comment:
Does it requires a small `in:fade` effect or it's smooth enough?

Reply:
Good idea, added.